qualang_tools/config/waveform_tools.py

```diff
@@ -42,7 +42,7 @@ def drag_gaussian_pulse_waveforms(
     z = gauss_wave + 1j * 0
     if alpha != 0:
         # The complex DRAG envelope:
-        z += 1j * gauss_der_wave * (alpha / (delta - detuning))
+        z += 1j * gauss_der_wave * (alpha / (delta - 2 * np.pi * detuning))
```

Comment:
I guess the 2pi should be in factor of (delta-detuning) because both delta and detuning are in Hz, right ?
Also I think we should replace detuning by anharmonicity whose meaning is clearer and closer to the referenced thesis and paper, what do you think?